server/src/LspServer.ts

```diff
@@ -295,9 +322,9 @@ export class LspServer {
     }
   }
 
-  async isDbtReady(): Promise<boolean> {
+  async isLanguageServerReady(): Promise<boolean> {
     try {
-      await this.dbtRpcServer.startDeferred.promise;
+      await Promise.all([this.dbtRpcServer.startDeferred.promise, this.contextFirstInitPromise]);
```

Comment:
Why we now need to await one more promise? If this.`contextFirstInitPromise` will be rejected the features will not work?

Reply:
Now we wait for context initialization to make it possible to create AST in opened docs. Otherwise we will need to propagate created context on all opened documents.